### PR TITLE
switch-to PostgreSQL 13 module during the upgrade

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

**Comment by evgeni** [comment]
The refactoring slightly escalated, sorry :D

File: definitions/scenarios/upgrade_to_capsule_6_16.rb [hunk 1]
```diff
@@ -41,6 +41,8 @@ class PreMigrations < Abstract
     def compose
       add_step(Procedures::Repositories::Setup.new(:version => '6.16'))
       if el8?
+        modules_to_switch = ['postgresql:13']
+        add_step(Procedures::Packages::SwitchModules.new(:module_names => modules_to_switch))
```
Comment (evgeni) [inline definitions/scenarios/upgrade_to_capsule_6_16.rb:45]: I pondered whether this should also be added to the upstream nightly upgrade definitions, and couldn't decide.

Comment (ehelms) [reply definitions/scenarios/upgrade_to_capsule_6_16.rb:45]: That seems correct -- is there an issue if it is added?

**Review by ehelms — APPROVED** [review]

File: definitions/scenarios/upgrade_to_capsule_6_16.rb (hunk 1 above)
Comment (evgeni) [reply definitions/scenarios/upgrade_to_capsule_6_16.rb:45]: Nah, switch-to to a module that is already enabled is a noop.
So you say "add"?

Comment (ehelms) [reply definitions/scenarios/upgrade_to_capsule_6_16.rb:45]: Yes

Comment (evgeni) [reply definitions/scenarios/upgrade_to_capsule_6_16.rb:45]: done

**Pull request merged**